lighthouse-core/config/config.js

```diff
@@ -145,6 +145,30 @@ function validatePasses(passes, audits, rootPath) {
   });
 }
 
+function validateCategories(categories, audits) {
+  if (!categories || !audits) {
```

Comment:
what about a `categories` with audits but no `audits` array? Wouldn't that fall under the `could not find audit` error?